gradle/pmd/customrulesset.xml

```diff
@@ -45,7 +45,8 @@
                     <![CDATA[
 //ThrowStatement
 //AllocationExpression
-[not(ClassOrInterfaceType[@Image='VersionInvalidException']) and not(ClassOrInterfaceType[@Image='ReadOnlyException'])]
+[not(ClassOrInterfaceType[@Image='VersionInvalidException']) and not(ClassOrInterfaceType[@Image='ReadOnlyException'])
+ and not (ClassOrInterfaceType[@Image= 'InvalidLicenseException'])]
```

Comment:
Shouldn't we rather have a nice message instead of adding an exception for this rule?

Reply:
are we finally keeping this? Is it needed?